README.md

```diff
@@ -125,9 +125,7 @@ on the above-mentioned Bioconda page.
 ## Usage - File download
 
 ```bash
-usage: pyega3 [-h] [-d] [-cf CONFIG_FILE] [-sf SERVER_FILE] [-c CONNECTIONS]
```

Comment:
I think this was formatted in this way, so that the README looks good (and readable) in GitHub, without scrolling.